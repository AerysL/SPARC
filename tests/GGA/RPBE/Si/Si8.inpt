# Test: Si8 #
CELL: 10.2 10.2 10.2
FD_GRID: 30 30 30
FD_ORDER: 12
BC: P P P
EXCHANGE_CORRELATION: GGA_RPBE
ELEC_TEMP_TYPE: fermi-dirac
ELEC_TEMP: 315.773
TOL_SCF: 1e-6
TOL_POISSON: 1e-8
TOL_PSEUDOCHARGE: 1e-8
NSTATES: 30
CALC_PRES: 1
CALC_STRESS: 1
