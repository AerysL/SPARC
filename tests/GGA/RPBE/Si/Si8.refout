***************************************************************************
*                           SPARC (May 11, 2020)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun May 17 23:27:39 2020                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.2 10.2 10.2 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 30 30 30
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.0009999935878
CHEB_DEGREE: 23
NSTATES: 30
EXCHANGE_CORRELATION: GGA_RPBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 1.16E-04
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: RPBE/Si/Si8
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 24
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 2 3 4
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  24
Mesh spacing                       :  0.34 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  RPBE/Si/Si8.out_1
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  RPBE/Si/Si.psp8
Pseudocharge radii of atom type 1  :  7.82 7.82 7.82 (x, y, z dir)
Number of atoms of type 1          :  8
Estimated total memory usage       :  42.85 MB
Estimated memory per processor     :  1.79 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9324126852E+00        2.405E-01        0.237
2            -3.9320835256E+00        1.823E-01        0.055
3            -3.9285790637E+00        3.596E-02        0.054
4            -3.9287923631E+00        2.054E-02        0.055
5            -3.9294195113E+00        6.293E-03        0.055
6            -3.9294377313E+00        3.074E-03        0.062
7            -3.9294496533E+00        1.248E-03        0.053
8            -3.9294499200E+00        5.125E-04        0.052
9            -3.9294499470E+00        2.560E-04        0.052
10           -3.9294499822E+00        4.716E-05        0.052
11           -3.9294499762E+00        2.920E-05        0.051
12           -3.9294500041E+00        1.141E-05        0.058
13           -3.9294499818E+00        6.310E-06        0.079
14           -3.9294499937E+00        2.021E-06        0.049
15           -3.9294500101E+00        1.182E-06        0.049
16           -3.9294499792E+00        5.577E-07        0.044
Total number of SCF: 16    

k = [0.000, 0.000, 0.000]
Occupation of state 27 = 0.000000000000000.
Occupation of state 30 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9294499804E+00 (Ha/atom)
Total free energy                  : -3.1435599843E+01 (Ha)
Band structure energy              :  1.6997423799E+00 (Ha)
Exchange correlation energy        : -9.8981444380E+00 (Ha)
Self and correction energy         : -1.0219506067E+02 (Ha)
Entropy*kb*T                       : -6.3633988266E-06 (Ha)
Fermi level                        :  2.4689177858E-01 (Ha)
RMS force                          :  1.8479994475E-02 (Ha/Bohr)
Maximum force                      :  4.7561811506E-02 (Ha/Bohr)
Time for force calculation         :  0.009 (sec)
Pressure                           :  1.0984999990E+01 (GPa)
Maximum stress                     :  1.1170188429E+01 (GPa)
Time for stress calculation        :  0.038 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  1.173 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
