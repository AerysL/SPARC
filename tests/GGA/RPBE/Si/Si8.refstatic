***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of Si:
      0.0100000000       0.0200000000       0.0250000000
      0.5000000000       0.5000000000       0.0000000000
      0.0000000000       0.5000000000       0.5000000000
      0.5000000000       0.0000000000       0.5000000000
      0.2500000000       0.2500000000       0.2500000000
      0.7500000000       0.7500000000       0.2500000000
      0.2500000000       0.7500000000       0.7500000000
      0.7500000000       0.2500000000       0.7500000000
Total free energy (Ha): -3.143559984291579E+01
Atomic forces (Ha/Bohr):
 -2.0070139382E-02  -2.6991057646E-02  -3.1562426288E-02
 -2.4905410335E-03  -3.5937691049E-03  -8.9280766258E-03
 -2.2461817892E-03  -3.4098306382E-03  -4.1500867814E-03
 -2.4435139597E-03  -6.7993228737E-03  -4.2710278289E-03
  2.7074459204E-02   2.7477316935E-02   2.7822592019E-02
  1.8133444006E-03   2.8919608952E-03   2.5669841534E-03
 -6.4071683091E-03   1.1169513877E-02   1.1918560681E-02
  4.7697408697E-03  -7.4481144372E-04   6.6034806711E-03
Stress (GPa): 
 -1.1170188429E+01  -1.9459467246E+00  -1.6773796406E+00 
 -1.9459467246E+00  -1.0975630509E+01  -1.2447880636E+00 
 -1.6773796406E+00  -1.2447880636E+00  -1.0809181032E+01
