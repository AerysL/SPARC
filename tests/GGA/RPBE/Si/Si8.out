***************************************************************************
*                           SPARC (Nov 06, 2019)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                 Start time: Thu Dec 19 15:14:57 2019                    *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.2 10.2 10.2 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 30 30 30
FD_ORDER: 12
BOUNDARY_CONDITION: 2
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 1
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.0009999932524
CHEB_DEGREE: 23
NSTATES: 30
EXCHANGE_CORRELATION: GGA_RPBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: Si8
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 25
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 1 5 5
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  25
Mesh spacing                       :  0.34 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  Si8.out
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  ./Si.psp8
Pseudocharge radii of atom type 1  :  8.16 8.16 8.16 (x, y, z dir)
Number of atoms of type 1          :  8
Fractional coordinates of atoms of type 1     :
      0.0100000000       0.0200000000       0.0250000000
      0.5000000000       0.5000000000       0.0000000000
      0.0000000000       0.5000000000       0.5000000000
      0.5000000000       0.0000000000       0.5000000000
      0.2500000000       0.2500000000       0.2500000000
      0.7500000000       0.7500000000       0.2500000000
      0.2500000000       0.7500000000       0.7500000000
      0.7500000000       0.2500000000       0.7500000000
========================================================================================
                    Self Consistent Field (SCF#1)                     
========================================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9245262746E+00        4.810E-02        0.898
2            -3.9283965485E+00        3.387E-02        0.232
3            -3.9291628687E+00        2.074E-02        0.235
4            -3.9294156735E+00        7.159E-03        0.238
5            -3.9294448591E+00        2.912E-03        0.235
6            -3.9294484965E+00        1.551E-03        0.236
7            -3.9294499095E+00        5.373E-04        0.234
8            -3.9294499562E+00        3.062E-04        0.226
9            -3.9294499718E+00        9.230E-05        0.225
10           -3.9294499743E+00        3.766E-05        0.219
11           -3.9294499867E+00        1.265E-05        0.209
12           -3.9294499736E+00        5.065E-06        0.211
13           -3.9294499806E+00        2.095E-06        0.207
14           -3.9294499624E+00        6.822E-07        0.208
Total number of SCF: 14    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9294499624E+00 (Ha/atom)
Total free energy                  : -3.1435599700E+01 (Ha)
Band structure energy              :  1.6997457374E+00 (Ha)
Exchange correlation energy        : -9.8981437656E+00 (Ha)
Self and correction energy         : -1.0219506067E+02 (Ha)
Entropy*kb*T                       : -6.3632334658E-06 (Ha)
Fermi level                        :  2.4689179899E-01 (Ha)
Atomic forces (Ha/Bohr):
 -2.0070113019E-02  -2.6991450330E-02  -3.1562865668E-02
 -2.4905520110E-03  -3.5939698126E-03  -8.9280011700E-03
 -2.2459719716E-03  -3.4100479008E-03  -4.1503859493E-03
 -2.4434537968E-03  -6.7993857849E-03  -4.2713205613E-03
  2.7074393635E-02   2.7477570390E-02   2.7822702126E-02
  1.8131858308E-03   2.8921495143E-03   2.5672373088E-03
 -6.4071134859E-03   1.1169590242E-02   1.1918807977E-02
  4.7696248182E-03  -7.4445631836E-04   6.6038259364E-03
RMS force                          :  1.8480210710E-02 (Ha/Bohr)
Maximum force                      :  4.7561985018E-02 (Ha/Bohr)
Time for force calculation         :  0.058 (sec)
Stress (GPa) : 
 -1.1170133619E+01  -1.9460094755E+00  -1.6774366674E+00 
 -1.9460094755E+00  -1.0975565034E+01  -1.2447777798E+00 
 -1.6774366674E+00  -1.2447777798E+00  -1.0809117098E+01 
Pressure                           :  1.0984938584E+01 (GPa)
Maximum stress                     :  1.1170133619E+01 (GPa)
Time for stress calculation        :  0.236 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  4.282 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
